deployment/hasura/metadata/databases/tables/actions/action_definition.yaml

```diff
@@ -0,0 +1,47 @@
+table:
+  name: action_definition
+  schema: actions
+configuration:
+  custom_name: "action_definition"
+object_relationships:
+  - name: workspace
+    using:
+      foreign_key_constraint_on: workspace_id
+  - name: uploaded_file
```

Comment:
This relationship should be renamed to something meaningful, for example, `action_file`.

Reply:
@AaronPlave Will this break a lot if we rename?

Reply:
Nope!